src/Resources/ClientLink.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Mollie\Api\Resources;
+
+use Mollie\Api\Types\ApprovalPrompt;
+
+class ClientLink extends BaseResource
+{
+    /**
+     * @var string
+     */
+    public $resource;
+
+    /**
+     * Id of the client link.
+     *
+     * @example csr_vZCnNQsV2UtfXxYifWKWH
+     * @var string
+     */
+    public $id;
+
+    /**
+     * An object with several URL objects relevant to the client link. Every URL object will contain an href and a type field.
+     *
+     * @var \stdClass
+     */
+    public $_links;
+
+    /**
+     * Get the redirect URL where the customer can complete the payment.
+     *
+     * @return string|null
+     */
+    public function getRedirectUrl(string $client_id, string $state, string $approval_prompt, array $scopes = [])
```

Comment:
We don't require `approval_prompt`, instead we have `auto` as default.

```suggestion
    public function getRedirectUrl(string $client_id, string $state, array $scopes = [], string $approval_prompt= 'auto')
```